aci/resource_aci_fvrsdomatt.go

```diff
@@ -644,6 +644,12 @@ func resourceAciDomainUpdate(ctx context.Context, d *schema.ResourceData, m inte
 	d.Partial(false)
 
 	if d.HasChange("enhanced_lag_policy") || d.HasChange("annotation") {
+		fvAEPgLagPolAtt := models.NewApplicationEPGLagPolicy(fmt.Sprintf(models.RnfvAEPgLagPolAtt), fvRsDomAtt.DistinguishedName, models.ApplicationEPGLagPolicyAttributes{})
```

Comment:
do we need the fmt.Sprintf in that. I thought models.RnfvAEPgLagPolAtt was already a string.

Reply:
Nope. Changed it.